source/plugins/ruby/CAdvisorMetricsAPIClient.rb

```diff
@@ -155,8 +155,8 @@ def getMetrics(winNode: nil, metricTime: Time.now.utc.iso8601)
           # Checking if we are in windows daemonset and sending only few metrics that are needed for MDM
           if !@os_type.nil? && !@os_type.empty? && @os_type.strip.casecmp("windows") == 0
             # Container metrics
-            metricDataItems.concat(getContainerMemoryMetricItems(metricInfo, hostName, "workingSetBytes", Constants::MEMORY_WORKING_SET_BYTES, metricTime, operatingSystem))
-            containerCpuUsageNanoSecondsRate = getContainerCpuMetricItemRate(metricInfo, hostName, "usageCoreNanoSeconds", Constants::CPU_USAGE_NANO_CORES, metricTime)
+            metricDataItems.concat(getContainerMemoryMetricItems(metricInfo, hostName, "workingSetBytes", Constants::MEMORY_WORKING_SET_BYTES, metricTime, operatingSystem, nameSpaceFilteringMode, nameSpaces))
+            containerCpuUsageNanoSecondsRate = getContainerCpuMetricItemRate(metricInfo, hostName, "usageCoreNanoSeconds", Constants::CPU_USAGE_NANO_CORES, metricTime, nameSpaceFilteringMode, nameSpaces)
```

Comment:
isn't similar filtering needed for getNodeMetricItemRate below?

Reply:
> isn't similar filtering needed for getNode* calls below?

> namespace not applicable for node related inventory and metrics since its namespaced object.